sotodlib/flags.py

```diff
@@ -83,3 +83,46 @@ def get_glitch_flags(tod, params={}, signal='signal', merge=True,
             tod.flags.wrap(name, flag)
         
     return flag
+
+def get_trending_flags(aman, max_trend=np.pi, n_pieces=1, signal_name='signal',
+                 flag_name='trends', overwrite=True):
+    """ Flag Detectors with trends larger than max_trend.
+    
+    Args:
+        aman (AxisManager): the tod 
+        max_trend: maxmium amount to always the detectors to change by
+                   default is pi for use in phase units.
```

Comment:
This could use a rewrite.  The second line here is not a new sentence -- sphinx will interpret it as a continuation of the first line.